for-upstream/gc.nix

```
@@ -14,9 +14,20 @@ in
         Amount of free space (GB) to ensure on garbage collection
       '';
     };
+    freespaceMinimumGB = mkOption {
+      type = types.nullOr types.int;
```

```suggestion
      type = types.int;
```

Not optional when freespaceGB isn't null.